OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -50,6 +50,9 @@ public class SpawnActorOnDeathInfo : ITraitInfo
 			"lead to unexpected behaviour.")]
 		public readonly CVec Offset = CVec.Zero;
 
+		[Desc("Defines if an actor is spawned even after the player was defeated (e.g. after surrendering).")]
+		public readonly bool SpawnAfterGameEnd = true;
```

Comment:
`SpawnAfterDefeat` 👍

I would reword the beginning of the desc to be `Should this actor spawn after the player...`.